x-pack/plugins/fleet/common/constants/mappings.ts

```diff
@@ -305,7 +305,6 @@ export const AGENT_MAPPINGS = {
           fields: {
             keyword: {
               type: 'keyword',
-              ignore_above: 16,
```

Comment:
Those fields where copied over from mappings but we don't need them here. Plus they were getting displayed in the searchboxes.